packages/vue-app/template/client.js

```diff
@@ -93,6 +93,7 @@ const errorHandler = Vue.config.errorHandler || console.error
 createApp()
   .then(mountApp)
   .catch((err) => {
```

Comment:
@pi0 Then `.catch(errorHandler)` is enough

Reply:
Yes. Agree. Cleaner code :)

Reply:
```js
createApp().then(mountApp).catch(errorHandler)
```

Reply:
Sure, makes sense :)